app/views/user_mailer/add_email.html.erb

```diff
@@ -36,7 +36,7 @@
 
 <p>
   <%= link_to add_email_confirmation_url(_request_id: @request_id, confirmation_token: @token, locale: @locale),
-              sign_up_create_email_confirmation_url(
+              add_email_confirmation_url(
```

Comment:
💡 what if we assigned `@add_email_url` in the mailer, and then re-used that variable here in the template instead of re-creating it 3 times, risking having different results? (I can also make a follow-up PR)

Reply:
Yep, I'm good with that

Reply:
https://github.com/18F/identity-idp/pull/7021